### PR TITLE
Add nzoueidi to Youtube and Zoom

### LINKED ISSUE #3889: REQUEST: New moderator for nzoueidi of Zoom and Youtube
### GitHub Username
@nzoueidi 

### Property you'd like to Moderate
Zoom, Youtube

### Username on Property Requested
nzoueidi

### Requirements

- [x] I am a Kubernetes Org member already
- [x] I have enabled 2FA on my account for the property I would like to moderate (if applicable).
- [x] I have read and will abide by the policies mentioned in the [Moderation Guidelines](https://git.k8s.io/community/communication/moderation.md)
- [x] I have two sponsors that meet the sponsor requirements listed in the moderator guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors

- @parispittman 
- @castrojo 

### List of qualifications for this position

- Part of the Ubuntu Board Membership which is a team that do meetings in order to approve new Ubuntu members. (https://launchpad.net/~ubuntu-membership-board/+members#active).
- Moderator of the mailing list of ubuntu-board-membership https://lists.ubuntu.com/mailman/private/ubuntu-membership-boards/
- Founder and organizer of Kubernetes and Cloud Native Tunisia meetup group< (https://www.meetup.com/Kubernetes-Cloud-Native-Tunisia/).
- Kubernetes coordinator.

### PR DESCRIPTION
**Pull request by nzoueidi**
Close #3889

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/lgtm
/hold
Just putting a hold on this till the action itself can be completed as a reminder. Adding members must be completed by the content owner using the google account associated with the administrator account.

/cc @castrojo @parispittman

**Comment by castrojo** [comment]
/approve

I don't think I have access to add the youtube admin priv though.

File: communication/moderators.md [hunk 1]
```diff
@@ -197,12 +198,13 @@ Moderators pro tempore seats: 10
 
 Administrators seats: 4
```
Comment (idealhack) [inline communication/moderators.md]: this needs an update too. how many admins do we need?

Comment (castrojo) [reply communication/moderators.md]: We can set that to whatever we want, the initial number was just an educated guess.

**Comment by alejandrox1** [comment]
think we need major permissions to add another manager to the account
/assign @parispittman

**Comment by parispittman** [comment]
invite sent for youtube 

@nzoueidi did other zoom admins give you the credentials there?

**Comment by alejandrox1** [comment]
ping @nzoueidi 
How is this going along?

**Comment by mrbobbytables** [comment]
It is done~


/hold cancel

**Pull request merged**